.github/workflows/fetch-jira-swaggers.yml

```diff
@@ -0,0 +1,49 @@
+name: Fetch Jira Swaggers
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: 0 0 * * *
+
+jobs:
+  fetch-swaggers:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checking out code
```

Comment:
nit: maybe align the wording with https://github.com/salto-io/adapter-swaggers/blob/main/.github/workflows/fetch-swaggers.yml ?